PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+## Please give the PR a descriptive title.
+
+## Type of change
+
```

Comment:
Is this naming scheme still in effect?:

https://wiki.fanatics.com/pages/viewpage.action?pageId=138234329